desktop-widgets/preferences/preferences_defaults.ui

```diff
@@ -35,18 +42,16 @@
       <item>
        <widget class="QFontComboBox" name="font"/>
       </item>
-
       <item>
        <widget class="QLabel" name="label_default1">
+        <property name="text">
+         <string/>
```

Comment:
I'm confused - what does this do? an empty label with a default name?

Reply:
That's QT tools decided to change order of properties in ui file. I'll fix this file manually.